nntrainer/tensor/tensor_v2.h

```diff
@@ -558,6 +558,56 @@ class TensorV2 {
   TensorV2 &multiply(TensorV2 const &m, TensorV2 &output,
                      const float beta = 0.0) const;
 
+  /**
+   * @brief     Add Tensor Element immediately to target tensor without mem copy
```

Comment:
immediately -> inplace ?

Reply:
not sure why the term "immediately" is used #131. I think "without mem copy" implies in-place computation though.